### DISCUSSION TITLE
How to change the background color of lua_ls message over the lower right corner in LazyVim?

### DISCUSSION THREAD

**Discussion by hopezh** [LazyVim/LazyVim#746] [Q&A]
I don't know what I have done that suddenly changes the background color of the lua_ls message shown at the lower right corner to this:

![2023-05-11 13 19 53](https://github.com/LazyVim/LazyVim/assets/23435810/7cfef302-504b-4b01-8c15-e5f90da18e88)

May I ask which config to edit to recover the original/default background color of this message area that came with LazyVima? 

Thanks.

**Comment by folke** [comment]
| HL | Default | Desc | 
-- | -- | --
NoiceFormatProgressDone | Search | Progress bar done
NoiceFormatProgressTodo | CursorLine | progress bar todo

**Reply by hopezh** [reply]
It seems that the sudden change of the background color of the lua_ls message happened when I'm using neovim in a tmux session. Not sure if it's related to the catppuccin-tmux theme I'm using...

**Comment by u3ih** [comment]
for me it was caused by the catppuccino theme for alacritty. at the end of the .yml file, there is a color configuration for index 16 inside indexed_colors. Just comment out that line
![image](https://github.com/LazyVim/LazyVim/assets/90165492/96bd0b51-10eb-48c7-b803-d975b473a528)
